src/features/filter-transactions/model.ts

```diff
@@ -1,6 +1,24 @@
-export type Status = 'committed' | 'rejected' | null;
-export type TabDefaultScreen = 'all' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
-export type TabBlocksScreen = 'transactions' | 'transfers' | 'mints' | 'burns' | 'grants' | 'revokes';
+import type { TransactionStatus } from '@/shared/api/schemas';
+
+export type Status = TransactionStatus | null;
+export type TabDefaultScreen = 'All' | 'Transfer' | 'Mint' | 'Burn' | 'Grant' | 'Revoke';
+export type TabBlocksScreen = 'Transactions' | 'Transfer' | 'Mint' | 'Burn' | 'Grant' | 'Revoke';
```

Comment:
I don't like this discrepancy between screens. It introduces unnecessary complications in code, and exists due to bad (_subjectively_) design.

More on that in the main comment of my review.